apps/desktop-wallet/src/api/transactions.ts

```diff
@@ -17,7 +17,7 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
 import { AddressHash, client } from '@alephium/shared'
-import { transactionSign } from '@alephium/web3'
+import { keyring } from '@alephium/shared-crypto'
```

Comment:
I named the class that holds all secrets `Keyring`, in accordance to Metamask's package: https://github.com/MetaMask/eth-hd-keyring/blob/main/index.js#L28